### PR TITLE
findCallsByName

### PR DESCRIPTION
**Pull request by scottfrazer**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
The more I see of this, the more convinced I am that winstanley should be calling into it. This is exactly the type of helper method the tutorial creates for "find-usages", "auto-completion", etc etc.

**Comment by cjllanwarne** [comment]
:+1:

File: src/main/scala/wdl4s/Workflow.scala [hunk 1]
```diff
@@ -133,6 +133,8 @@ case class Workflow(unqualifiedName: String,
     (callInputs ++ declarationInputs) map { input => input.fqn -> input } toMap
   }
 
+  def findCallByName(name: String): Option[Call] = calls.find(_.unqualifiedName == name)
```
Comment (mcovarr) [inline src/main/scala/wdl4s/Workflow.scala]: doc that the param is an lqn

**Comment by mcovarr** [comment]
:+1:

**Pull request merged**